src/backend/tools/web_scrape.py

```diff
@@ -18,7 +18,14 @@ async def call(self, parameters: dict, **kwargs: Any) -> List[Dict[str, Any]]:
 
         response = get(url)
         if not response.ok:
-            return Exception(f"Cannot open and scrape URL {url}")
+            return [
+                (
+                    {
+                        "text": f"Cannot open and scrape URL {url}",
```

Comment:
last nit: can you add the reason or the error message to the text as well?

Reply:
oop yes!